src/tests/NavBar.react.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(
+      <Router basename={process.env.PUBLIC_URL}>
+        <Link to="/" active="true" exact="true">
+          Rockets
```

Comment:
- [ ] This doesn't qualify as a test. you should import a component and write a test for it.
- [ ] Kindly write tests that cover a significant number of components.
- [ ] Apart from snapshot tests you are expected to also write unit tests.

Reply:
Understood. :+1: